templates/edge/partials/left-nav.md.hbs

```diff
@@ -117,7 +119,6 @@
 	<h2>Instance properties</h2>
 	<ul>
 		<li>{{{createLink 'ractive.container'}}}</li>
-		<li>{{{createLink 'ractive.nodes'}}}</li>
 		<li>{{{createLink 'ractive.transitions (instance)' 'ractive.transitions'}}}</li>
 		<li>{{{createLink 'ractive.parent'}}}</li>
 		<li>{{{createLink 'ractive.partials (instance)' 'ractive.partials'}}}</li>
```

Comment:
readthedocs solved this problem by creating a menu based on directory structure if a menu structure wasn't defined in the YAML file.